lib/msal-node/src/cache/Storage.ts

```diff
@@ -49,80 +39,118 @@ export class Storage extends CacheManager {
         this.changeEmitters.forEach(func => func.call(null));
     }
 
+    /**
+     * Converts cacheKVStore to InMemoryCache
+     * @param cache
+     */
+    cacheToInMemoryCache(cache: CacheKVStore): InMemoryCache {
+
+        let inMemoryCache: InMemoryCache = {
+            accounts: {},
+            idTokens: {},
+            accessTokens: {},
+            refreshTokens: {},
+            appMetadata: {},
+        };
+
+        for (let key in cache) {
+            if (cache[key] instanceof AccountEntity) {
+                inMemoryCache.accounts[key] = cache[key] as AccountEntity;
+            } else if (cache[key] instanceof IdTokenEntity) {
+                inMemoryCache.idTokens[key] = cache[key] as IdTokenEntity;
+            } else if (cache[key] instanceof AccessTokenEntity) {
+                inMemoryCache.accessTokens[key] = cache[key] as AccessTokenEntity;
+            } else if (cache[key] instanceof RefreshTokenEntity) {
+                inMemoryCache.refreshTokens[key] = cache[key] as RefreshTokenEntity;
+            } else if (cache[key] instanceof AppMetadataEntity) {
+                inMemoryCache.appMetadata[key] = cache[key] as AppMetadataEntity;
+            } else {
+                continue;
+            }
+        }
+
+        return inMemoryCache;
+    }
+
+    /**
+     * converts inMemoryCache to CacheKVStore
+     * @param inMemoryCache
+     */
+    inMemoryCacheToCache(inMemoryCache: InMemoryCache): CacheKVStore {
```

Comment:
Note: I need to test this, verify if this over writes existing values with same keys. Will test and change as needed. `spread` operator ideally should, but since we are pushing the data to `Record<string, object>`, need to test all use cases.